tests/dashboard/build-dashboard.test.ts

```diff
@@ -117,12 +117,14 @@ describe('GitHub Discussions Processing', () => {
   });
 
   it('should handle complete failure', async () => {
+    delete process.env.GITHUB_TOKEN; // Ensure token is missing
```

Comment:
Don't delete the env variable directly in tests. Store the initial in a temporary variable, and then manipulate the env value during tests.